src/simulation.py

```diff
@@ -8,14 +8,15 @@
 class Simulation():
     def __init__(self):
         self.config = Config()
+        self.generation_duration = self.config.generation_duration
         self.world = World(self.config)
         self.population = Population(self.config)
         self.screen = screen_init(self.config)
```

Comment:
Would it work to just replace this with the self.reset() function? (I'm not sure because it's the constructor, but I think it should work and would prevent repetition).
```suggestion
        self.reset()
```